Externals/Nimble

```diff
@@ -1 +1 @@
-Subproject commit 0cf12916a276ba7bc7a66ded95bd8f7e77ca7f6b
+Subproject commit f24eac653b32a20b6c0f1d92c964ddd1adadfffc
```

Comment:
I think this might also be unnecessary. I don't think the pending changes would require an update to the version of Nimble being used...?